zetta_utils/geometry/bbox_strider.py

```diff
@@ -240,7 +240,7 @@ def shape(self) -> Vec3D[int]:  # pragma: no cover
 
     def get_all_chunk_bboxes(self) -> List[BBox3D]:
         """Get all of the chunks."""
-        if self.num_chunks > multiprocessing.cpu_count():
+        if self.num_chunks > 100 * multiprocessing.cpu_count():
```

Comment:
The `100*cpu_count` threshold doesn't really make sense - you're just increasing the threshold (penalizing?) larger head nodes with more CPUs than smaller ones.

I think it makes more sense as a threshold that does not depend on machine's cpu count, so:
```
TASK_THRESHOLD = 4*128  // pick a reasonable number that would be slow for a single thread to execute
...
if self.num_chunks > TASK_THRESHOLD:
   with multiprocessing.Pool() as pool_obj:
...
```

Reply:
Reasonable - the overhead should be mostly constant.

Reply:
Ah if you worry about overhead, you can limit the number of processes spawned, based on chunk count:
```
n_procs = self.num_chunks // 128  # 128 chunks per proc, or however much
n_procs = min(multiprocessing.cpu_count(), n_procs)  # don't spawn more than cpu count
with multiprocessing.Pool(n_procs) as pool_obj:
...
```